### ISSUE TITLE
Gradle tests no longer running

### ISSUE THREAD

**Issue by allredj** [halo-dev/halo#1127]
Hi!

It seems that since this commit: https://github.com/halo-dev/halo/commit/11f48781cb6321fa27a21907aeec14e2d6570373
the gradle tests can no longer be run using the `./gradlew test` command.
Is this expected?

Many thanks for your reply.
Joel

**Comment by allredj** [comment]
Actually I was able to make the tests run by re-introducing
```
test {
    useJUnitPlatform()
}
```
in the `build.gradle`.
Shall I submit a PR?

**Comment by JohnNiang** [comment]
We are using JUnit5 instead of JUnit.

**Comment by ruibaby** [comment]
Any other exchanges?

**Comment by allredj** [comment]
I'm no Gradle expert, but I think that the `useJUnitPlatform()` is also required for JUnit5: https://www.baeldung.com/junit-5-gradle

**Comment by JohnNiang** [comment]
Yup, Same tip is at https://docs.gradle.org/current/userguide/java_testing.html#using_junit5.

But I can run the gradle test using gradle wrapper.

```java
╰─$ ./gradlew test                                                          

Welcome to Gradle 6.3!

Here are the highlights of this release:
 - Java 14 support
 - Improved error messages for unexpected failures

For more details see https://docs.gradle.org/6.3/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)

> Task :test
2020-11-02 06:15:36.870  INFO 3626551 --- [extShutdownHook] o.s.s.concurrent.ThreadPoolTaskExecutor  : Shutting down ExecutorService 'applicationTaskExecutor'
2020-11-02 06:15:36.870  INFO 3626551 --- [extShutdownHook] o.s.s.c.ThreadPoolTaskScheduler          : Shutting down ExecutorService 'taskScheduler'
2020-11-02 06:15:36.889 DEBUG 3626551 --- [extShutdownHook] run.halo.app.cache.InMemoryCacheStore    : Cancelling all timer tasks
2020-11-02 06:15:36.889  INFO 3626551 --- [extShutdownHook] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
Hibernate: drop table attachments if exists
2020-11-02 06:15:36.893 ERROR 3626551 --- [extShutdownHook] o.h.engine.jdbc.spi.SqlExceptionHelper   : Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
2020-11-02 06:15:36.894 ERROR 3626551 --- [extShutdownHook] o.h.engine.jdbc.spi.SqlExceptionHelper   : Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
2020-11-02 06:15:36.894  WARN 3626551 --- [extShutdownHook] o.s.b.f.support.DisposableBeanAdapter    : Invocation of destroy method failed on bean with name 'entityManagerFactory': org.hibernate.exception.GenericJDBCException: Unable to release JDBC Connection used for DDL execution
2020-11-02 06:15:36.894  INFO 3626551 --- [extShutdownHook] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
2020-11-02 06:15:36.896  INFO 3626551 --- [extShutdownHook] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.

BUILD SUCCESSFUL in 25s
6 actionable tasks: 2 executed, 4 up-to-date
```

**Comment by JohnNiang** [comment]
Maybe you could submit a PR for this issue.

**Comment by allredj** [comment]
Here's the PR: https://github.com/halo-dev/halo/pull/1130.
I suppose CI will fail because there are failing tests.

**Comment by allredj** [comment]
Tests are fixed, and #1130 is merged. I verified locally that the tests were now running with `./gradlew test`. Thank you for the quick turnaround! :+1:

**Comment by lhqypp** [comment]
![image](https://user-images.githubusercontent.com/103261938/162611586-da53d31f-fc02-44da-8100-85259e2fec2c.png)
什么都没有啊
